File: SIAN/Bcasein.jl
    # Muñoz-Tamayo, R., de Groot, J., Bakx, E., Wierenga, P.A., Gruppen, H., Zwietering, M.H., Sijtsma, L., 2011. 
    # Hydrolysis of β-casein by the cell-envelope-located PI-type protease of Lactococcus lactis: A modelling approach.
    # Int. Dairy J. 21. https://doi.org/10.1016/j.idairyj.2011.03.012
    
using SIAN

ode = @ODEmodel(
   x1'(t)  = -k*x1(t)/(Km*(1+(10-x1(t))/kI)+x1(t)),
   y1(t)   = x1(t)
   );

output = identifiability_ode(ode, get_parameters(ode));

File: GenSSI 2.0/Bcasein.m
    % Muñoz-Tamayo, R., de Groot, J., Bakx, E., Wierenga, P.A., Gruppen, H., Zwietering, M.H., Sijtsma, L., 2011. 
    % Hydrolysis of B-casein by the cell-envelope-located PI-type protease of Lactococcus lactis: A modelling approach.
    % Int. Dairy J. 21. https://doi.org/10.1016/j.idairyj.2011.03.012
    
function model = Basein()


    % Symbolic variables
     syms x1 
     syms k Km kI
        
     x0 = 10; % Initial concentrationg of B-casein
     I = x0 - x1;
    
    
    % Parameters
    
    model.sym.p = [k; Km; kI];
 
     % State variables
    model.sym.x = [x1];
    
    % Functions 
     
     roh   = k*x1/(Km*(1+I/kI)+x1); % Competitive inhibition kinetics
               
     
    % Control vectors (g)
    model.sym.g = [];
    
    % Autonomous dynamics (f)
                 
                         
       model.sym.xdot=[ -roh];

    % Initial conditions
     model.sym.x0 = [x0]; 
    
     % Observables
     model.sym.y = [x1];
end
